server/Mistakes.Journal.Api/Api/Mistakes/WebModels/WeatherWebModel.cs

```diff
@@ -0,0 +1,36 @@
+namespace Mistakes.Journal.Api.Api.Mistakes.WebModels
```

Comment:
Could you create different namespace for Weather? I think it has nothing to do with mistakes.

Reply:
We've established I'll organize it later